lib/workos/organizations/organizations.ex

```diff
@@ -0,0 +1,99 @@
+defmodule WorkOS.Organizations do 
+  import WorkOS.API 
+
+  @moduledoc """
+  The Organizations module provides resource methods for working with Organizations
+  """
+
+  @doc """
+  Create an organization
+
+  ### Parameters
```

Comment:
Should we also support the idempotency key here, like in the Ruby SDK? https://github.com/workos/workos-ruby/blob/main/lib/workos/organizations.rb#L85

Reply:
Interesting, what's an idempotency key? I didn't find it in the API reference https://workos.com/docs/reference/organization/create

Reply:
Strange, I guess we have a separate section on Idempotency: https://workos.com/docs/reference/idempotency

Little blurb:

```
The WorkOS API supports idempotency which guarantees that performing the same operation multiple times will have the same result as if the operation were performed only once. This is handy in situations where you may need to retry a request due to a failure or prevent accidental duplicate requests from creating more than one resource.
```

Very helpful for these create requests, so a duplicate org isn't created. You can take it as an optional parameter, and if present, you can send it as a request header.

Reply:
We can also add this as a follow-up

Reply:
I think we can add it here already!

Reply:
Nice!